samcli/lib/build/app_builder.py

```diff
@@ -187,17 +188,24 @@ def _build_layer(self, layer_name, codeuri, runtime):
             manifest_path = self._manifest_path_override or os.path.join(code_dir, config.manifest_name)
 
             # By default prefer to build in-process for speed
+            build_runtime = specified_workflow
             build_method = self._build_function_in_process
             if self._container_manager:
-                build_method = self._build_function_on_container
+                build_method = self._build_function_in_process
+                if config.language == "provided":
+                    LOG.warning(
```

Comment:
Nit: should we say which layer we are trying to build in here? or is it already covered elsewhere?

Reply:
Everytime a resource is build we see a log `Building layer 'LayerOne'` or `Building function 'FunctionOne'`.